src/components/Navbar.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { AiOutlineLeft } from 'react-icons/ai';
+import { BsFillGearFill } from 'react-icons/bs';
+import { Link } from 'react-router-dom';
+import './styles/Navbar.css';
+
+const Navbar = () => {
+  return (
+    <nav className="navbar-container">
+      <div>
+        <div>
+          <ul className="navbar">
+            <li>
+              <Link to="/" className="home"> <AiOutlineLeft /> </Link>
```

Comment:
- [ ] Kindly ensure that the _back_ navigation button is not shown/ on the _homepage_. This button performs no function on the _homepage_.